substrate/frame/state-trie-migration/src/lib.rs

```diff
@@ -601,7 +655,14 @@ pub mod pallet {
 
 			// ensure they can pay more than the fee.
 			let deposit = T::SignedDepositPerItem::get().saturating_mul(limits.item.into());
-			ensure!(T::Currency::can_slash(&who, deposit), Error::<T>::NotEnoughFunds);
+			sp_std::if_std! {
+				use frame_support::traits::tokens::{Fortitude, Preservation};
+				println!("+ {:?} / {:?} / {:?}", who, deposit, T::Currency::reducible_balance(&who, Preservation::Protect, Fortitude::Force));
```

Comment:
one `println!` was there before in `migrate_custom_child` extrinsic, so I just added the same to others,
there is a `sp_std::if_std! {` so it should not affect runtime and I suppose that somebody used it for tests or benchmarks or something like that.
But I think, no problem to remove those.

Reply:
It might be useful to still get them in logs at the debug level. I've recently been setting the log level to debug and it's very helpful, maybe try something like: 
```rust
log::trace!("+ {:?} / {:?} / {:?}", who, deposit, T::Currency::reducible_balance(&who, Preservation::Protect, Fortitude::Force));
```
Otherwise yeah just remove.